src/docs/pages/component-docs/components/three-liner/three-liner-doc.module.ts

```diff
@@ -1,16 +1,27 @@
 import { NgModule } from '@angular/core';
 import { BasicExample } from './examples/basic.component';
 import { SharedCompDocsModule } from '../../shared/shared-comp-docs.module';
-import { ChannelValueModule, ThreeLinerModule } from '@brightlayer-ui/angular-components';
+import { ChannelValueModule, ThreeLinerModule, ToolbarMenuModule } from '@brightlayer-ui/angular-components';
```

Comment:
So it looks like you replaced channel value with toolbar menu, which is fine by me. Would you want to take `ChannelValueModule` out of the import statement?

Reply:
Good catch, it's removed now.